tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -31,9 +31,11 @@ jobs:
       displayName: 'Post code coverage data to the Dashboard'
       inputs:
         script: |
-          python3 $(Build.SourcesDirectory)/tools/ci_build/github/windows/post_code_coverage_to_dashboard.py  --commit_hash=$(OnnxRuntimeGitCommitHash) --report_file="$(Build.BinariesDirectory)/report.json" --report_url="https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=$(Build.BuildId)"
+          python3 $(Build.SourcesDirectory)/tools/ci_build/github/windows/post_code_coverage_to_dashboard.py  --commit_hash=$(OnnxRuntimeGitCommitHash) --report_file="$(Build.BinariesDirectory)/report.json" --report_url="https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=$(Build.BuildId)"
         workingDirectory: $(Build.BinariesDirectory)
       env: 
         DASHBOARD_MYSQL_ORT_PASSWORD: $(dashboard-mysql-ort-password)
 
-    - template: templates/clean-agent-build-directory-step.yml
+    - task: mspremier.PostBuildCleanup.PostBuildCleanup-task.PostBuildCleanup@3
```

Comment:
why not using the template for cleaning? doing cleaning by hand seems error prone

Reply:
This task has a special property, it always runs after all other normal build steps. 
So, when the devops system auto injects something into our build job,
these things will run before this "PostBuildCleanup". If I don't change this line, it will be after. In that case, the injected task can't find the files they were looking for.